### PR TITLE
Change dependencies

### PR DESCRIPTION
**Pull request by trackleft**
This worked for me.

### CODE REVIEW

File: modules/custom/az_core/config/install/linkit.linkit_profile.az_linkit.yml [hunk 1]
```diff
@@ -2,11 +2,14 @@ langcode: en
...
   config:
-    - image.style.linkit_result_thumbnail
+    - image.style.az_tiny_thumbnail
   module:
     - file
     - image
     - node
+  enforced:
+    module:
+      az_core
```
Comment (trackleft) [inline modules/custom/az_core/config/install/linkit.linkit_profile.az_linkit.yml:12]:
```suggestion
      - az_core
```

File: modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+uuid: 69404e6f-b69a-4aff-8940-9a53baefc304
+langcode: en
+status: true
+dependencies:
+  module:
+    - file
+    - image
+  enforced:
+    module:
+      az_core
```
Comment (trackleft) [inline modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml:10]:
```suggestion
      - az_core
```

File: modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+uuid: 69404e6f-b69a-4aff-8940-9a53baefc304
```
Comment (trackleft) [inline modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml:1]:
```suggestion
```

**Review by joeparsons — APPROVED** [review]
Approved (pending the commit suggestion getting committed)

**Comment by joeparsons** [comment]
This should be good to merge into @akslay's PR branch.

**Comment by akslay** [comment]
This PR seems to install cleanly and LinkIt functionality is working as expected. Once we merge this in to mine, we will still need to figure out the metadata for the LinkIt results before we merge into main.

**Review by akslay — APPROVED** [review]

**Pull request merged**